playbook/app/pb_kits/playbook/pb_background/docs/_background_light.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
 import { Background } from '../..'
+import styles from './style.scss'
 
 const BackgroundLight = (props) => (
   <Background
       backgroundColor="light"
       padding="xl"
       {...props}
+      className={styles.classy}
```

Comment:
Do you want to keep these in production...or just for testing?

Reply:
These were just for testing. I will delete